src/Haskell/Ide/Engine/Transport/LspStdio.hs

```diff
@@ -41,6 +41,7 @@ import qualified GhcModCore               as GM ( loadMappedFileSource, getMMapp
 import           Haskell.Ide.Engine.Config
 import qualified Haskell.Ide.Engine.Ghc   as HIE
 import           Haskell.Ide.Engine.LSP.CodeActions
+import           Haskell.Ide.Engine.LSP.Completions
```

Comment:
It would be nice if you imported this qualified. I personally find it easier to read when using the functions

Reply:
@lorenzo this should be *only* a code move.  If we want to change the code that moved, that should be a separate PR.

Otherwise it is too hard to follow the trail of what happened. In my opinion.

Reply:
Well, the code was being used qualified before this move, so I think keeping the style is desirable.

Reply:
Added it back!